v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/SourceDbToSpannerOptions.java

```diff
@@ -71,6 +71,7 @@ public interface SourceDbToSpannerOptions extends CommonTemplateOptions {
   /* TODO: (support Sharding, PG namespaces) */
   @TemplateParameter.Text(
       order = 5,
+      regexes = {"(^[^\\/?%*:|\"<>.]{1,64}$)"},
```

Comment:
This regex is too expensive for dataflow. 
`ERROR: (gcloud.dataflow.flex-template.run) FAILED_PRECONDITION: (406846f638a637e8): Template metadata regex '(^[^\/?%*:|"<>.]{1,64}$)' was too large. Max size is 1000 but was 1031.`